tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+
+SKIP_CLUSTER_START=true
+export SKIP_CLUSTER_START
+
+TEST_HELPER_DIR="$BATS_TEST_DIRNAME/../test_helper"
+export TEST_HELPER_DIR
+
+load "$TEST_HELPER_DIR"/tools_check.bash
+load "$TEST_HELPER_DIR"/fluvio_dev.bash
+load "$TEST_HELPER_DIR"/bats-support/load.bash
+load "$TEST_HELPER_DIR"/bats-assert/load.bash
+
+setup_file() {
+  echo "No-Op"
+}
+
+@test "Install fvm and setup a settings.toml file" {
+    run $FVM_BIN self install
+    assert_success
+
+    # Ensure the `~/.fvm/` directory is available
+    test -d ~/.fvm
+    assert_success
+
+    # Ensure the `fvm` binary is available
+    test -f ~/.fvm/bin/fvm
+    assert_success
+
+    # Ensure the `settings.toml` is available
+    test -f ~/.fvm/settings.toml
+    assert_success
```

Comment:
Much better.  Still need to check if setting is correct

Reply:
At this point is an empty file, given that no version is installed already.
As of today `toml` doesn't support `null`/`nil` it just leaves the file empty.

Reply:
After FVM is installed, should be assumed that the channel is set to stable

Reply:
Or just not set at all. let channel manager set

Reply:
I think we should leave that to install.sh and a test around that script to default to stable, this is self install and shouldn't have side effect logic.